typescript/sdk/src/core/EvmCoreReader.ts

```diff
@@ -77,4 +82,19 @@ export class EvmCoreReader implements CoreReader {
 
     return results as CoreConfig;
   }
+
+  private async _getProxyAdminConfig(
```

Comment:
not the biggest fan of `_` in front of private functions. i know we do it in solidity, but lets keep it without, unless you want to update the other private functions to keep consistency :)

Reply:
I might be willing to change all private functions to use `_` 🤣 but I'll change it to keep the project consistent. Old habits coming back ahah.